index.html

```diff
@@ -48,6 +48,30 @@ <h2>Connect</h2>
             <p>github: <a href="https://github.com/marice-romero">github.com/marice-romero</a></p>
         </section>
 
+        <section>
+            <h2>Leave a Message</h2>
+            <form name="leave_message">
```

Comment:
In the javascript file you try to reference this form by using getElementById(), while the form only has the name, which gives an error in the console and breaks functionality.
You should give the form and id attribute as well -> <form name="leave_message" id="leave_message">

Reply:
Hi Mina! Thanks so much for checking - I'm seeing an id in my local VS Code, but it didn't upload for some reason. I might have forgotten to save it?